data/themes/darktable.css

```diff
@@ -594,7 +612,8 @@ overshoot.right
 
 .dt_module_focus #iop-panel-label /* set focus mode */
 {
-  color: @plugin_focus_color;
+  color: shade(@plugin_label_color, 1.1);
+  text-shadow: 0px 0px 2px shade(@plugin_label_color, 1.05);
```

Comment:
I suppose this is the culprit :)

Reply:
@elstoc and @TurboGit: I've tried on this one to set the label as possible near to the new highlight focus icon but as said on first post, not totally convinced by the result. So you're probably right to not let that. Anyway, the previous version lack of contrast. So could you test with that change before I merge it:

```
.dt_module_focus #iop-panel-label/* set focus mode */
{
  color: shade(@plugin_label_color, 1.2);
}
```
Waiting for your feedback.

Reply:
Looks good to me.